submissions/zalevskyi/html-css-popup/css/popup-input.css

```diff
@@ -0,0 +1,27 @@
+/*Menu setup*/
+#menu_recycle_checkbox {
```

Comment:
Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkboxes in right way.